review/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/resource/PassengerResourceTestIT.java

```diff
@@ -0,0 +1,146 @@
+package cz.muni.fi.airportmanager.passengerservice.resource;
+
+import cz.muni.fi.airportmanager.passengerservice.entity.Passenger;
+import cz.muni.fi.airportmanager.passengerservice.model.CreatePassengerDto;
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusIntegrationTest
+@TestHTTPEndpoint(PassengerResource.class)
+class PassengerResourceTestIT {
+
+    String savePassenger(CreatePassengerDto passengerDto) {
+        return given().contentType("application/json")
+                .body(passengerDto)
+                .when()
+                .post()
+                .then()
+                .statusCode(201)
+                .extract()
+                .response()
+                .path("id")
+                .toString();
+    }
+
+    @AfterEach // this method will be called after each test
+    void tearDown() {
+        given().when()
+                .delete()
+                .then()
+                .statusCode(200);
+    }
+
+    @Test
+    void shouldSaveAndGetListOfPassengers() {
+        // Take this as an example for the rest of the tests
+        CreatePassengerDto testPassenger = createTestPassengerDto();
+        var id = savePassenger(testPassenger);
+
+        given().when()
+                .get("/" + id)
+                .then()
+                .statusCode(200)
+                .body("email", is(testPassenger.email));
+    }
+
+    @Test
+    void shouldSaveAndGetEmptyListOfPassengers() {
+        // TODO implement this test
+        given().when()
+                .get()
+                .then()
+                .statusCode(200)
+                .body(is("[]"));
+    }
+
+    @Test
+    void shouldGetAndGetPassenger() {
+        // TODO implement this test
+        CreatePassengerDto testPassenger = createTestPassengerDto();
+        savePassenger(testPassenger);
+
+        given().when()
+                .get()
+                .then()
+                .statusCode(200)
+                .body("size()", is(1));
+    }
+
+
+
+    @Test
+    void shouldDeleteAllPassengers() {
```

Comment:
here it would be good to first create some passengers because you are clearing everything after each test and verify that that get all is empty